packages/frontend-v2/utils/apiRouting.ts

```diff
@@ -0,0 +1,19 @@
+import { GetStudentResponse } from "@graduate/common";
+import { NextRouter } from "next/router";
+
+export const routeOnboarding = (
```

Comment:
Not sure if this name/file name is correct. Any suggestions for what I should rename this/where I should place this?